opencsp/common/lib/geometry/test/test_Vxy.py

```diff
@@ -1,18 +1,27 @@
+import os
+import unittest
+
 import matplotlib.pyplot as plt
 import numpy as np
 
 from opencsp.common.lib.geometry.Vxy import Vxy
 import opencsp.common.lib.render.figure_management as fm
+import opencsp.common.lib.tool.file_tools as ft
+
 
+class TestVxy(unittest.TestCase):
+    def setUp(self) -> None:
+        path, _, _ = ft.path_components(__file__)
+        self.data_dir = os.path.join(path, "data", "input", "Vxy")
+        self.out_dir = os.path.join(path, "data", "output", "Vxy")
+        ft.create_directories_if_necessary(self.data_dir)
+        ft.create_directories_if_necessary(self.out_dir)
 
-class TestVxy:
-    @classmethod
-    def setup_class(cls):
-        cls.V1 = Vxy((2, 2))
-        cls.V1_array = np.array([[2], [2]])
+        self.V1 = Vxy((2, 2))
+        self.V1_array = np.array([[2], [2]])
 
-        cls.V2 = Vxy((1, 2))
-        cls.V2_array = np.array([[1], [2]])
+        self.V2 = Vxy((1, 2))

```

Comment:
Does this generate a random vector? If not, I would recommend using randomly sized and seeded vectors.

Reply:
I like this idea. Done!